src/index.html

```diff
@@ -1,13 +1,367 @@
+          <h2 class="subscription__title title">Підписка</h2>
+
+          <p class="subscription__text">
+            Бути в курсі всіх актуальних подій та останніх новин.
+          </p>
+
+          <form
+            action="#"
+            method="post"
+            class="subscription__form"
+          >
+            <input
+              type="email"
+              name="email"
+              class="subscription__email"
+              placeholder="E-mail"
+              required
+            >
+
+            <button type="submit" class="subscription__submit"></button>
+          </form>
+        </div>
+      </section>
+    </main>
+
+    <footer class="footer" id="contacts">
+      <div class="container container--footer">
+        <div class="footer__top">
+          <div class="footer__top-logo">
+            <a href="#home" class="footer__top-link">
+              NA <br> MU
+            </a>
+          </div>
+
+          <div class="footer__top-social">
+            <a
+              href="https://uk-ua.facebook.com/"
+              class="footer__facebook"
+              target="_blank"
+            ></a>
+            <a
+              href="https://www.instagram.com/"
+              class="footer__instagram"
+              target="_blank"
+            ></a>
+          </div>
+        </div>
+
+        <div class="footer__info">
+          <div class="footer__info-block-1">
+            <div class="footer__schedule timetable">
+              <h3 class="timetable__title">Графік роботи</h3>
+
+              <ul class="timetable__list">
+                <li class="timetable__day">
+                  <strong>12:00 — 20:00</strong> середа
+                </li>
+                <li class="timetable__day">
+                  <strong>12:00 — 20:00</strong> четвер
+                </li>
+                <li class="timetable__day">
+                  <strong>12:00 — 20:00</strong> п’ятниця
+                </li>
+                <li class="timetable__day">
+                  <strong>11:00 — 19:00</strong> субота
+                </li>
+                <li class="timetable__day">
+                  <strong>11:00 — 19:00</strong> неділя
+                </li>
+              </ul>
+
+              <div class="timetable__weekend">
+                Вихідні: понеділок, вівторок
+                <br>
+                Каса припиняє роботу за 1 годину до закриття музею
+              </div>
+            </div>
+
+            <div class="footer__info-contacts">
+              <div class="footer__info-contacts-name">Контакти:</div>
+
+              <div class="footer__contacts">
+                <a
+                  href="https://goo.gl/maps/8ECKZaozX3ZxoGvw9"
+                  class="footer__info-address"
+                  target="_blank"
+                >
+                  вул. М. Грушевського, 6; Київ, 01001
+                </a>
+
+                <a href="tel: +38 (044) 278-13-57" class="footer__info-tel">
```

Comment:
Need to fix format phone only in href everywhere
```suggestion
                <a href="tel: +380442781357" class="footer__info-tel">
```